src/components/Menu/Menu.jsx

```diff
@@ -8,15 +8,16 @@ export class Menu extends React.Component {
     return (
       <div className="menu">
         <h1>Menu</h1>
-        {sandwiches.map((sandwich, i) => {
-          const { name, price } = sandwich;
+        {sandwiches.map((category, i) => {
```

Comment:
I don't think `sandwiches` is the right term anymore for the data as its now split into categories. It feels like its mapping over sandwiches but its actually mapping over menu categories